packages/apps/docs/src/docs/deploy/chainweb/compact-a-database.md

```diff
@@ -1,103 +1,124 @@
 menu: Deploy
-label: Compact a Chainweb node database
+label: Compact Chainweb databases
 order: 2
 layout: full
 tags: [pact, chainweb, network, node operator, developer]
 
 ---
-# Compact a Chainweb node database
-
-## This feature is unstable. Make sure to back up your database before using this tool.
+# Compact Chainweb node databases
 
 Because a healthy blockchain continuously adds new transactions in new blocks that change the state of the database, managing the storage requirements on individual nodes can be challenging.
 
-To address this storage issue, Chainweb provides the `cwtool compact` command.
-The `cwtool compact` command enables you to delete historical unused state from the `chainweb-node` Pact SQLite database.
+To address this storage issue, Chainweb provides the `compact` command-line program.
+The `compact` command enables you to delete historical unused state from the `chainweb-node` RocksDB database and the Pact SQLite database.
 Removing old state that isn't required to validate transactions or reach consensus enables your node to use far less disk space overall while maintaining the semantic integrity of node operations.
 
+After you compact the state and restart the node to use the compacted database, you can delete the old database to further reduce your storage overhead or save the old database in another location as a backup.
+
 To compact a Chainweb node database:
 
 1. Open a terminal shell on a computer with access to the `chainweb-node` you manage.
 
    For example, if you run the node in a Docker container, open a terminal in the container.
-   If you installed chainweb-node from a release binary or built it from source, open a terminal or secure shell on the computer where the binary is installed.
-
-   For information about installing chainweb-node from a release binary or building it from source code, see the [chainweb-node README](https://github.com/kadena-io/chainweb-node#README).
+   If you installed `chainweb-node` from a release binary or built it from source, open a terminal or secure shell on the computer where the binary is installed.
 
-2. Verify that you have access to the `cwtool`  command-line program by running the following command:
+   For information about installing `chainweb-node` from a release binary or building it from the source code, see the [chainweb-node README](https://github.com/kadena-io/chainweb-node#README).
 
-   ```bash
-   cwtool --help
-   ```
+   If you run the node in a Docker container, pull the latest image to download the `compact` binary.
+   If you built the node from the source code, you can rebuild to get the `compact` binary.
 
-   If you have access to the `cwtool` program, you should see usage information similar to the following:
+2. Verify that you have access to the `compact` command-line program by running the following command:
 
    ```bash
-   Chainweb Tool
+   compact --help
+   ```
 
```

Comment:
This seems like kind of a weird way to say this. If you're in the docker container you want `compact --help`, but otherwise if you changed directory to where your chainweb-node binary is, you want `./compact --help`. And if you have it installed globally to your machine, you still want `compact --help`. This makes me wonder if we should rename to `chainweb-compact` or something, but I guess that's besides the point.

Reply:
I had to do some wrangling to get both chainweb-node and compact binaries living together (compact wasn't included in the release package until this week and I had to pull a special version of the docker image to see both binaries).
Maybe I can just delete those extra instructions.

Reply:
Yeah might be best, seeing as the regular docker image doesn't seem to have `compact`, unless @chessai changed that.